aws/rust-runtime/aws-types/src/config.rs

```diff
@@ -60,6 +64,40 @@ impl Builder {
         self
     }
 
+    /// Set the retry_config for the builder
+    ///
+    /// # Examples
+    /// ```rust
+    /// use aws_types::config::Config;
+    /// use smithy_types::retry::RetryConfig;
+    /// let config = Config::builder().retry_config(RetryConfig::new()).build();
+    /// ```
+    pub fn retry_config(mut self, retry_config: RetryConfig) -> Self {
```

Comment:
We should make `region` and `set_region` consistent with these, but it shouldn't be part of this PR. Could you file an issue labeled with `breaking-change` in smithy-rs to follow up on this?

Reply:
Done https://github.com/awslabs/smithy-rs/issues/748